include/lbann/utils/file_utils.hpp

```diff
@@ -65,7 +65,7 @@ bool create_dir(const std::string output_dir);
 
 
 /// Load a file into a buffer
-template <typename T = std::vector<unsigned char> >
+template <typename T = std::vector<char> >
```

Comment:
Do we need this template?

Reply:
No, the previous version of code made use of the template, but we don't need it.